Fabric.Identity.API/Management/UsersController.cs

```diff
@@ -36,7 +43,7 @@ public UsersController(IDocumentDbService documentDbService, AbstractValidator<U
                     HttpStatusCode.NotFound);

```

Comment:
Should we use the CreateFailureResponse here so we can  tell consumers of the API that we are returning a 404 because the client they requested did not exist? As it is now, it will be hard for them to know if it's the client id or the document ids that caused the 404.

Reply:
I cant see it in the view here but this is the client Id check 

`if (string.IsNullOrEmpty(client?.ClientId))
            {
                return CreateFailureResponse($"The specified client with id: {clientId} was not found",
                    HttpStatusCode.NotFound);
            }`